torchaudio/csrc/ffmpeg/stream_reader/buffer/common.cpp

```diff
@@ -128,6 +138,25 @@ std::tuple<torch::Tensor, bool> get_image_buffer(
 
   int height = frame->height;
   int width = frame->width;
+
+  switch (fmt) {
+    case AV_PIX_FMT_RGB48LE: {
+      auto buffer = get_buffer16({num_frames, height, width, channels}, device);
+      return std::make_tuple(buffer, false);
+    }
+#ifdef USE_CUDA
+    case AV_PIX_FMT_CUDA: {
+      AVHWFramesContext* hwctx = (AVHWFramesContext*)frame->hw_frames_ctx->data;
+      AVPixelFormat sw_format = hwctx->sw_format;
+      switch (sw_format) {
+        case AV_PIX_FMT_P010: {
+          auto buffer = get_buffer16({num_frames, channels, height, width}, device);
+          return std::make_tuple(buffer, true);
+        }
+      }
+    }
+#endif
+  }
```

Comment:
Can you add `default:` label so that compiler won't issue warning on missing branches?